lib/db.js

```diff
@@ -453,16 +496,26 @@ Db.prototype.createCollection = deprecateOptions(
  * @param {Db~resultCallback} [callback] The collection result callback
  * @return {Promise} returns Promise if no callback passed
  */
+const statsSchema = {
+  readPreference: { type: [ReadPreference, 'object', 'string'] },
```

Comment:
looks like this extra `object` is making it in everywhere, was there a test that required this? We only document `ReadPreference|string`